includes/adminpages.php

```diff
@@ -246,6 +246,30 @@ function pmpro_admin_bar_menu() {
 			)
 		);
 	}
+
+	// Add menu item for adding a new member.
+	if ( current_user_can( 'manage_options' ) ) {
```

Comment:
Should this include membership manager role? A more specific capability here?

Reply:
I'm going to mark this resolved as we will do a full "capabilities" conversation at a later time and come up with a plugin-wide ruleset and update then if necessary.